plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -0,0 +1,2589 @@
+        """
+
+        try:
+            response = self.dnac._exec(
+                family="event_management",
+                function='get_email_destination'
+            )
+            self.log("Received API response from 'get_email_destination': {0}".format(str(response)), "DEBUG")
+
+            if not response:
+                self.log("There is no Email destination present in Cisco Catalyst Center", "INFO")
+                return response
+
+            return response[0]
+
+        except Exception as e:
+            self.status = "failed"
+            self.msg = "Error while getting the details of Email destination present in Cisco Catalyst Center: {0}".format(str(e))
+            self.log(self.msg, "ERROR")
+            self.check_return_status()
+
+    def collect_email_playbook_params(self, email_details):
+        """
+        Collects the parameters required for configuring Email destinations from the playbook.
+        Args:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+            email_details (dict): A dictionary containing the Email destination details from the playbook.
+        Returns:
+            dict: A dictionary containing the collected parameters for configuring Email destinations.
+        Description:
+            This function collects the parameters required for configuring Email destinations from the playbook.
+            It extracts parameters such as 'fromEmail', 'toEmail', 'subject', and SMTP configurations
+            (primary and secondary) from the provided email_details dictionary.
+        """
+
+        playbook_params = {
+            'fromEmail': email_details.get('from_email'),
+            'toEmail': email_details.get('to_email'),
+            'subject': email_details.get('subject')
+        }
+
+        if email_details.get('primary_smtp_config'):
+            primary_smtp_details = email_details.get('primary_smtp_config')
+            playbook_params['primarySMTPConfig'] = {}
+            playbook_params['primarySMTPConfig']['hostName'] = primary_smtp_details.get('hostname')
+            playbook_params['primarySMTPConfig']['port'] = primary_smtp_details.get('port', "25")
+            playbook_params['primarySMTPConfig']['smtpType'] = primary_smtp_details.get('smtp_type', "DEFAULT")
+            playbook_params['primarySMTPConfig']['userName'] = primary_smtp_details.get('username', '')
+            playbook_params['primarySMTPConfig']['password'] = primary_smtp_details.get('password', '')
+
+        if email_details.get('seconday_smtp_config'):
+            secondary_smtp_details = email_details.get('secondary_smtp_config')
+            playbook_params['secondarySMTPConfig'] = {}
+            playbook_params['secondarySMTPConfig']['hostName'] = secondary_smtp_details.get('hostname')
+            playbook_params['secondarySMTPConfig']['port'] = secondary_smtp_details.get('port')
```

Comment:
same as above

Reply:
dded the port number to 25 if snmp_type is "DEFAULT", for other we have the option from 465 and 587 based on type(SSL, TLS)